tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,30 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin: ""
+  vin_list: ""
+  ble_mac: ""
+  ble_mac_list: ""
+  presence_detection_ttl: 180
   debug: false
   mqtt_ip: ""
   mqtt_port: "1883"
   mqtt_user: ""
   mqtt_pwd: ""
   send_cmd_retry_delay: "5"
+
 schema:
-  vin1: str?
-  vin2: str?
-  vin3: str?
-  presence_detection: bool
-  ble_mac1: str?
-  ble_mac2: str?
-  ble_mac3: str?
+  vin: str?
+  vin_list: "match(^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)"
```

Comment:
Need to be tested

Reply:
Are you sure to want to force a format? I have never tested this kind of core in HA. Will test tomorrow

Reply:
First the doc is [here](https://developers.home-assistant.io/docs/add-ons/configuration/)

Well... if the regex is good, all it can do is help end user ensure he inputs 17 chars for the VIN and a proper MAC address.

I had ask ChatGPT to generate those 2 I ask again and specify this is for HA, it came with a slightly different result:
```
Generic: (^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)
Python : (^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)
```
Per everything I tried, there was 1 answer with a double \\ instead of single `(^([A-HJ-NPR-Z0-9]{17})(\\|[A-HJ-NPR-Z0-9]{17})*$)`

Reply:
Tested with a Python script, this works.

```
import re

vin_pattern = re.compile(r'^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$')

vin_test_1 = "1HGCM82633A123456"
vin_test_2 = "1HGCM82633A123456|1HGCM82633A123457"

# Check VINs
print(bool(vin_pattern.match(vin_test_1)))  # True
print(bool(vin_pattern.match(vin_test_2)))  # True
```

When removing or adding 1 char, it fails.